sphinx-docs/usermanual/userman_admin.rst

```diff
@@ -204,30 +218,101 @@ Downloading Individual Videos
 #. View subtopics by clicking on the '+' symbol to the left of a subject of your choice. You can cose them by clicking on the '-' symbol.
 #. Mark the content you wish to download by clicking the checkbox to the left of the content name. 
 #. Click the green "Download" button in the top left box of the page. The button should also show you the total number of videos you have selected to download, as well as the total size of the content.
-#. Once the download is completed, video content will be ready for students to watch!
+#. Once the download is completed, video content will be ready for Learners to watch!
 
 Downloading Videos in Bulk
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If you want to download videos in bulk, your best option is to download the KA Lite videos through the `BitTorrent Sync`_ client. This will be a much faster process than using the KA Lite app to download all of the videos.
+
+We have made the full set of KA videos (in the format needed by KA Lite) available via `BitTorrent Sync`_ (btsync). Note that this is different from BitTorrent; btsync allows us to add new videos or fix problems without issuing a whole new torrent file, and then having seeders split between the old and new torrent files. Here are the steps to set this up:
+
+#. Download and install `BitTorrent Sync`_.
+#. Run btsync. On some platforms, this will bring up a graphical interface. On Linux, you will need to load http://127.0.0.1:8888/ to get the interface
```

Comment:
Maybe it should read `you will need to type http://127.0.0.1:8888/ in to the address bar of your browser to get the interface`